tests/h/views/badge_test.py

```diff
@@ -25,6 +25,32 @@ def test_badge_returns_number_from_search(models, search_run):
     assert result == {'total': 29}
 
 
+@badge_fixtures
+def test_badge_returns_number_from_search_if_unauth_user(models, search_run):
```

Comment:
The logic which fetches the count doesn't differ depending on whether the user is authenticated or not, only the logic which calls NR to log a custom event. Is there a reason you're not testing the calls to NR here?